src/windows/export.py

```diff
@@ -468,77 +485,83 @@ def cboSimpleTarget_index_changed(self, widget, index):
-
-                        # get the audio bit rates
-                        audiobitrate = xmldoc.getElementsByTagName("audiobitrate")
-                        for audiorate in audiobitrate:
-                            a_l = audiorate.attributes["low"].value
-                            a_m = audiorate.attributes["med"].value
-                            a_h = audiorate.attributes["high"].value
-                            self.abr = {_("Low"): a_l, _("Med"): a_m, _("High"): a_h}
-
-                        # get the remaining values
-                        vf = xmldoc.getElementsByTagName("videoformat")
-                        self.txtVideoFormat.setText(vf[0].childNodes[0].data)
-                        vc = xmldoc.getElementsByTagName("videocodec")
-                        self.txtVideoCodec.setText(vc[0].childNodes[0].data)
-                        sr = xmldoc.getElementsByTagName("samplerate")
-                        self.txtSampleRate.setValue(int(sr[0].childNodes[0].data))
-                        c = xmldoc.getElementsByTagName("audiochannels")
-                        self.txtChannels.setValue(int(c[0].childNodes[0].data))
-                        c = xmldoc.getElementsByTagName("audiochannellayout")
-
-                        # check for compatible audio codec
-                        ac = xmldoc.getElementsByTagName("audiocodec")
-                        audio_codec_name = ac[0].childNodes[0].data
-                        if audio_codec_name == "aac":
-                            # Determine which version of AAC encoder is available
-                            if openshot.FFmpegWriter.IsValidCodec("libfaac"):
-                                self.txtAudioCodec.setText("libfaac")
-                            elif openshot.FFmpegWriter.IsValidCodec("libvo_aacenc"):
-                                self.txtAudioCodec.setText("libvo_aacenc")
-                            elif openshot.FFmpegWriter.IsValidCodec("aac"):
-                                self.txtAudioCodec.setText("aac")
+            for preset_folder in [info.EXPORT_PRESETS_PATH, info.USER_PRESETS_PATH]:
+                for file in os.listdir(preset_folder):
+                    preset_path = os.path.join(preset_folder, file)
+                    try:
+                        xmldoc = xml.parse(preset_path)
+                        title = xmldoc.getElementsByTagName("title")
+                        if _(title[0].childNodes[0].data) == selected_target:
+                            profiles = xmldoc.getElementsByTagName("projectprofile")
+
+                            # get the basic profile
+                            all_profiles = False
+                            if profiles:
+                                # if profiles are defined, show them
+                                for profile in profiles:
+                                    profiles_list.append(_(profile.childNodes[0].data))
```

Comment:
```suggestion
                                profiles_list.extend([_(p.childNodes[0].data) for p in profiles])
```

Reply:
Second time you masking `append` to `extend` This isn't good.